tests/requirements.txt

```diff
@@ -0,0 +1,7 @@
+testinfra>=3,<4
+pytest-xdist==1.28.0
+pytest>=4.6,<5.0
+ansible>=2.10,<2.11,!=2.9.10
+Jinja2>=2.10
+jmespath
```

Comment:
We only need `jmespath` if we were using the `json_query` filter in Ansible, and we're not using it here, so we can drop this.